frontend/ui/widget/inputtext.lua

```diff
@@ -402,10 +403,6 @@ function InputText:initTextBox(text, char_added)
                 end
                 self:setText(self:getText(), true)
             end,
-
-            padding = self.padding,
-            margin = self.margin,
-            bordersize = self.bordersize,
```

Comment:
A bit confusing, someone added it once for a reason I'm sure?

Reply:
That would be you in #3088 ;p.

Reply:
I must've had a reason, though it could always be an accidental failure to clean up some thought earlier in the process. :-P

Reply:
It also was in the checkbutton
https://github.com/koreader/koreader/blob/04d1d23c2f6207a74d447c5633ad07740bb6df81/frontend/ui/widget/checkbutton.lua#L44-L46
but not used anywhere further in the code of the module.

Now checkbutton is a pure checkbutton and we place it exactly where we want in the parent widget (eg new `Case sensitive` in the search). Open with... and Show password do the same accurate placement.